gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/oneconvergence_service_node_driver.py

```diff
@@ -62,7 +57,7 @@
 
 cfg.CONF.register_opts(oneconvergence_driver_opts, "oneconvergence_node_driver")
 
-SC_METADATA = '{"sc_instance":"%s", "insert_type":"%s", "floating_ip": "%s", "provider_interface_mac": "%s"}'
+SC_METADATA = '{"sc_instance":"%s", "floating_ip": "%s", "provider_interface_mac": "%s"}'
 SVC_MGMT_PTG_NAME = cfg.CONF.oneconvergence_node_driver.svc_management_ptg_name
 STITCHING_PTG_NAME = "traffic-stitching-gbp-internal"
```

Comment:
The code is not ready for review yet, thats why did not assign the team to the pull request :)